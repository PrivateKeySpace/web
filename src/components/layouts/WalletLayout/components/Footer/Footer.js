import React from 'react'
import { compose, setDisplayName, pure } from 'recompose'
import { css } from 'aphrodite/no-important'
import tx from '../../../../../theme/styles'
import sx from './styles'

const displayName = 'components/layouts/WalletLayout/components/Footer'

const enhance = compose(
  setDisplayName(displayName),
  pure
)

function Footer () {
  return (
    <footer className={css(tx.px5)}>
      <div className={css(tx.width100, tx.py4, tx.px1, tx.textTransformUppercase, sx.footer)}>
        2018 &copy; PKS Wallet. All rights reserved.
      </div>
    </footer>
  )
}

export default enhance(Footer)
