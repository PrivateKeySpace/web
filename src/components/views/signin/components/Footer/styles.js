import { StyleSheet } from 'aphrodite/no-important'
import { colors } from '../../../../../theme/constants'

const styles = StyleSheet.create({
  footerContainer: {
    marginTop: 'auto'
  },
  footer: {
    borderTopWidth: 1,
    borderTopStyle: 'solid',
    borderTopColor: colors.lightGray,
    color: colors.lightGray
  }
})

export default styles
