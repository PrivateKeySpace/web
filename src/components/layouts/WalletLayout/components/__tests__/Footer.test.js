import React from 'react'
import { StyleSheetTestUtils } from 'aphrodite'
import { render } from 'enzyme'
import Footer from '../Footer'

beforeEach(() => {
  StyleSheetTestUtils.suppressStyleInjection()
})

afterEach(() => {
  StyleSheetTestUtils.clearBufferAndResumeStyleInjection()
})

it('renders without crashing and matches snapshot', () => {
  const footer = render(
    <Footer />
  )
  expect(footer).toMatchSnapshot()
})
