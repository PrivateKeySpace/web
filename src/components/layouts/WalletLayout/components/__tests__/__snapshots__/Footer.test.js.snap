// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing and matches snapshot 1`] = `
<footer
  class="px5_1g0n6pg-o_O-footerContainer_14ddvtv"
>
  <div
    class="width100_zdxht7-o_O-py4_r2hutn-o_O-px1_17oj4c4-o_O-textTransformUppercase_4efw5a-o_O-footer_r1wilx"
  >
    2018 Â© PKS Wallet. All rights reserved.
  </div>
</footer>
`;
